src/main/kotlin/com/ampnet/userservice/persistence/repository/UserInfoRepository.kt

```diff
@@ -3,15 +3,14 @@ package com.ampnet.userservice.persistence.repository
 import com.ampnet.userservice.persistence.model.UserInfo
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.data.jpa.repository.Query
-import java.util.Optional
 import java.util.UUID
 
 interface UserInfoRepository : JpaRepository<UserInfo, UUID> {
-    fun findBySessionId(sessionId: String): Optional<UserInfo>
+    fun findBySessionIdOrderByCreatedAtDesc(sessionId: String): List<UserInfo>
```

Comment:
Is this correct? How can there be multiple UserInfo rows with the same sessionId?

Reply:
Because `sessionId` is generated by the KYC provider. The problem occurred when Veriff sent user data for the same user and also for the session. Veriff handles session internally and it can be reused for multiple situations in KYC process.